lib/OctokitAction.ts

```diff
@@ -30,6 +34,22 @@ export abstract class OctokitAction extends Action {
     return this.graphqlWithAuth(query);
   }
 
+  protected async moveIssue(issueId: string, transitionName: string): Promise<void> {
+    try {
+      console.log(`${issueId}: Getting transition id for '${transitionName}'`);
+      let response = await this.jiraTransitionRequest("GET", issueId);
+      const transition = response.transitions.find((t: any) => t.name === transitionName);
+      if (transition == null) {
+        throw new Error(`Could not find the transition '${transitionName}'`);
```

Comment:
Exception driven logic is bad. Log directly instead. And the logging should be inside the `listTransitions` method, so we don't need to reimplement it every time

Reply:
Finding the transition should not be part of `listTransitions`; I could extract this logic in a `findTransition` method.